luxonis_ml/data/datasets/luxonis_dataset.py

```diff
@@ -404,15 +413,22 @@ def get_classes(
 
         return classes, classes_by_task
 
-    def get_skeletons(self) -> Dict[str, Dict]:
+    def get_skeletons(self, sync_mode: bool = False) -> Dict[str, Dict]:
         """Returns the dictionary defining the semantic skeleton for each class using
         keypoints.
 
         @rtype: Dict[str, Dict]
         @return: A dictionary mapping classes to their skeleton definitions.
         """
+        if sync_mode:
+            local_file = osp.join(self.metadata_path, "skeletons.json")
+            self.fs.get_file("metadata/skeletons.json", local_file)
+            with open(local_file) as file:
+                skeletons = json.load(file)
```

Comment:
Should we be returning some warning there if there is no file?

Reply:
Yes, it would be beneficial to include a warning here.